File: JJB/231127/행렬의_덧셈.py
def solution(arr1, arr2):
    answer = []
    
    for col1, col2 in zip(arr1, arr2):
        temp = []
        for row1, row2 in zip(col1, col2):
            temp.append(row1 + row2)
        answer.append(temp)
    
    return answer

File: JJB/Programmers/231127/행렬의_덧셈.cpp
#include <string>
#include <vector>

using namespace std;

vector<vector<int>> solution(vector<vector<int>> arr1, vector<vector<int>> arr2) 
{
    vector<vector<int>> answer;
    
    size_t cols_size = arr1.size();
    size_t rows_size = arr1.at(0).size();
    
    for(int col = 0; col < cols_size; col++)
    {
        vector<int> temp;
        for(int row = 0; row < rows_size; row++)
            temp.push_back(arr1.at(col).at(row) + arr2.at(col).at(row));
        answer.push_back(temp);
    }
    
    return answer;
}